
FAIL query() returns dictionary with empty arrays when no locks are held promise_test: Unhandled rejection with value: object "TypeError: undefined is not an object (evaluating 'navigator.locks.query')"

